server/src/main/java/org/apache/iotdb/db/auth/AuthorityChecker.java

```diff
@@ -138,6 +140,7 @@ private static int translateToPermissionId(Operator.OperatorType type) {
       case INSERT:
       case LOADDATA:
       case CREATE_INDEX:
+      case ALTER_TIMESERIES:
         return PrivilegeType.INSERT_TIMESERIES.ordinal();
```

Comment:
This should be pointed out in the document that altering a timeseries requires a write privilege.

Reply:
Can you tell me the place that I can mention it?